src/backend/database_migrations/versions/20210503_184213_pangolin_probability_to_float.py

```diff
@@ -0,0 +1,34 @@
+"""pangolin probability to float
+
+Create Date: 2021-05-03 18:42:14.746783
+
+"""
+import enumtables  # noqa: F401
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20210503_184213"
+down_revision = "20210427_171625"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.alter_column(
+        "pathogen_genomes",
+        "pangolin_probability",
+        existing_type=sa.Integer(),
+        type_=sa.Float(),
+        schema="aspen",
+    )
+
+
+def downgrade():
+    op.alter_column(
+        "pathogen_genomes",
+        "pangolin_probability",
+        existing_type=sa.Float(),
+        type_=sa.Integer(),
+        schema="aspen",
+    )
```

Comment:
we should just prohibit a downgrade, as casting a float from 0-1 to an int is pretty meaningless.

Reply:
ok i added a `NotImplementedError` for when downgrading the DB